src/generic-container/started-generic-container.ts

```diff
@@ -100,8 +111,8 @@ export class StartedGenericContainer implements StartedTestContainer {
     return this.inspectResult.networkSettings[networkName].ipAddress;
   }
 
-  public exec(command: Command[]): Promise<ExecResult> {
-    return execContainer(this.container, command);
+  public exec(command: Command[], options: Partial<ExecOptions> = {}): Promise<ExecResult> {
+    return this.execContainer(command, options);
```

Comment:
could you please inline this private method? having 2 functions called `execContainer` is a bit confusing